einhorn.gemspec

```diff
@@ -23,5 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'chalk-rake'
   gem.add_development_dependency 'subprocess'
 
+  gem.add_runtime_dependency 'ffi'
```

Comment:
`ffi` was the one I found examples and documentation for. I did not know about `fiddle`, and from my quick attempts, `fiddle` is horribly, horribly documented. For Einhorn specifically it's probably worth sticking to stdlib... I'll try to make it work.